src/sentry/static/sentry/app/components/platformPicker.jsx

```diff
@@ -170,6 +170,8 @@ const SearchBar = styled('div')`
     background: none;
     padding: 2px 4px;
     width: 100%;
+    // Ensure a consistent line height to keep the input the desired height
```

Comment:
```suggestion
    /* Ensure a consistent line height to keep the input the desired height */
```
Don't comments in CSS tags in JSX need block comments?

Reply:
Yes they do.

Reply:
Is this supposed to be updated to be a block comment?

Reply:
Thanks, not sure why it didn't apply eralier.

Reply:
I think you force pushed and overwrote the commit github made.

Reply:
Yeah you're right